common/djangoapps/student/admin.py

```diff
@@ -162,9 +165,51 @@ class UserProfileInline(admin.StackedInline):
     verbose_name_plural = _('User profile')
 
 
+class EmailValidationMixin(object):
+    def clean_email(self):
+        """
+        Checks that email is not blank and there is no user with such email yet.
+        If initial is provided, it is treated as the previous value of the email.
+        So if the email did not change, there is no need to check the DB for duplicate emails.
+        """
+        email = self.cleaned_data['email']
+        initial = self.initial.get('email', None)
+        if not email or not email.strip():
+            raise ValidationError("Email can not be blank.", code='blank')
+        elif initial != email and User.objects.filter(email=email).exists():
+            raise ValidationError("User with such email already exists.", code='unique')
+        return email
+
+
+class UserCreationForm(BaseUserCreationForm, EmailValidationMixin):
+    """ New user creation form. Makes sure the email is unique and not blank. """
+
+
+class UserChangeForm(BaseUserChangeForm, EmailValidationMixin):
+    """ New user creation form. Makes sure the email is unique and not blank. """
```

Comment:
the same as above, and change title please)

Reply:
done